### ISSUE TITLE
🖊️⚙️ Implement Input and Validation for Nonce and Gas Tip in Custom TX Parameters

### ISSUE THREAD

**Issue by vkjr** [status-im/status-react#21024]
# Feature

Implement Nonce edit flow: https://www.figma.com/design/xLs1KYmF4e6WwRTZVJKeUK/Wallet?node-id=17671-228120&t=BUk6LX8HGnOk60xD-4
Implement Gas Amount flow: https://www.figma.com/design/xLs1KYmF4e6WwRTZVJKeUK/Wallet?node-id=17671-229653&t=BUk6LX8HGnOk60xD-4

## TBD
- where to get texts?

## Acceptance Criteria

- [ ] The help section should be visible
- [ ] Error messages in case of invalid nonce or gas amount

**Comment by churik** [comment]
I do not think that it is feasible to include this feature to  2.32, postponing due to limited capacity in order to keep release scope clear

cc @shivekkhurana

**Comment by churik** [comment]
postponing it for now to 2.34, cc @shivekkhurana
